src/main/resources/jenkins/branch/DeadBranchIndicatorAction/jobMain.jelly

```diff
@@ -0,0 +1,43 @@
+<!--
+The MIT License
+
+Copyright (c) 2019 CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+    <!--TODO build-caption i.e. the links to the builds on the job page doesn't take effect-->
+    <style type="text/css">
```

Comment:
How does `jobMain.jelly` get loaded? I would've thought it would need to be named `action.jelly` to be picked up [here](https://github.com/jenkinsci/jenkins/blob/82aca6421da3c4f889d79a34197157928d5c4436/core/src/main/resources/lib/hudson/actions.jelly#L39), but I am probably missing something.

Reply:
* `action.jelly` is for rendering in the left sidebar
* `jobMain.jelly` is for [rendering in the center main page on a job](https://github.com/jenkinsci/workflow-job-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowJob/main.jelly#L39), like the last junit results, warnings etc.
* `summary.jelly` is for [rendering in the center main page of a build](https://github.com/jenkinsci/workflow-job-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowRun/index.jelly#L63) like the junit or warnings summary etc.

They are standard action views across both `AbstractBuild`s and `WorkflowRun`s.

Reply:
There is also [floatingBox.jelly](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/hudson/project/projectActionFloatingBox.jelly#L38) which renders the trend graphs on the right hand side of a build page.